packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -63,6 +63,46 @@ const defaultProps = {
   hideAccessoryDisplays: false,
 };
 
+const createConcpetDisplays = (compareConceptDisplays) => {
+  if (compareConceptDisplays.systolic && compareConceptDisplays.diastolic) {
+    if (compareConceptDisplays.systolic === compareConceptDisplays.diastolic) {
+      return <div className={cx('concept-display')}>{compareConceptDisplays.originalSystolic}</div>;
+    }
+    return (
+      <div className={cx('concept-display')}>
+        {compareConceptDisplays.originalSystolic}
+        {' / '}
+        {compareConceptDisplays.originalDiastolic}
+      </div>
+    );
+  }
+  if (compareConceptDisplays.systolic || compareConceptDisplays.diastolic) {
+    const conceptDisplayValue = compareConceptDisplays.originalSystolic || compareConceptDisplays.originalDiastolic;
+    return <div className={cx('concept-display')}>{conceptDisplayValue}</div>;
+  }
+  return null;
+};
+
+const createDatetimeDisplays = (compareDatetimeDisplays) => {
+  if (compareDatetimeDisplays.systolic && compareDatetimeDisplays.diastolic) {
+    if (compareDatetimeDisplays.systolic === compareDatetimeDisplays.diastolic) {
+      return <div className={cx('concept-display')}>{compareDatetimeDisplays.originalSystolic}</div>;
+    }
+    return (
+      <div className={cx('concept-display')}>
+        {compareDatetimeDisplays.originalSystolic}
+        {' / '}
+        {compareDatetimeDisplays.originalDiastolic}
+      </div>
+    );
+  }
+  if (compareDatetimeDisplays.systolic || compareDatetimeDisplays.diastolic) {
+    const conceptDisplayValue1 = compareDatetimeDisplays.originalSystolic || compareDatetimeDisplays.originalDiastolic;
```

Comment:
again, this seems like it would be hard for the user to differentiate when value they are looking at if one was missing. They wouldn't know which was missing

Reply:
Thanks @emilyrohrbough - I will take this as a followup to check with clinical owners and what they expect to see, if the only importance is if there is a difference, or strong desire to know which label/concept is associated with which display.